update to 0.6.1

dayne opened the pull request.

dayne committed:
update to 0.6.0

dayne committed:
Update default.rb

jasonblalock commented:
I'm not 100% sure but I'm pretty sure you need to update the checksum as well so it matches 0.6.0. I believe `3cc90846ca972d88b601789af2ad9ed0a496447a13cb986a3d74a4de062af37d` is the proper checksum. 

Were you able to run test kitchen successfully after upping the version? Mine failed, throwing a git error, which I thought was related to `install_with_make` running `make` and `make install`. Specifically `make` for ruby-install tries to run a git command, which of course won't work because this isn't a git directory.

rosstimson commented:
Thanks for the PR @dayne 

I would indeed expect the shasum to change too.

Also, this test:  https://github.com/rosstimson/chef-ruby_install/blob/master/test/integration/installation/serverspec/installation_spec.rb#L15

davejlong commented:
Has the reason that this PR not been merged only that the checksum needs to be updated? The current checksum for ruby-install 0.6.0 is `3cc90846ca972d88b601789af2ad9ed0a496447a13cb986a3d74a4de062af37d`.

dayne committed:
update to 0.6.1

cleaning out comments.

dayne committed:
update test kitchen platforms

dayne committed:
update tests and specs for install & partial

these converge and verify successfully.

dayne committed:
proposed: rev version and updated CHANGELOG

dayne commented:
@rosstimson 

Since submitting this pull request I've leveled up on understanding the test kitchen and important that part of a pull request is.

I've come back to this pull request and updated it to 0.6.1, correct checksum, and updated test kitchen environment to ensure this cookbook is still relevant on modern systems and updated rubies.

The  `installation` and `partial-ruby-version` test suites converge and verify cleanly.

kitchen suite `alltherubies` converges on older platforms but has issues with rbx and jruby parts.  Looking at your README it appears you are aware of this issue. I dug in a bit and it appears `ruby-install` itself would need to be updated to support modern platforms such as Ubuntu-16.10 to allow for successful building and testing of rbx and jruby.  

Thanks

dayne committed:
update alltherubies suite to modern version

note: jruby and rbx do not really work on modern platforms.
ruby-install would need to be updated it appears.
